src/index.js

```diff
@@ -109,7 +110,18 @@ class App extends React.Component {
                 let trashDay = mySubResponse.feature.attributes.TRASHDAY;
                 this.updateInfo(trashDay);
             });
-        });
+        })
+		.catch((error) => { // if no address/KIVA is found
+			console.log("Cannot find Kiva");
+			this.setState({
+				gotData: false
+			});
+			
+			document.body.style.cursor = "default"; // changes the cursor to the normal pointer to singal the user that nothing is loading
+			alert("Please make sure the address you are entering is in the Kansas City area and not in a surrounding area like North Kansas City or Liberty.");
+		});
+		console.log("Kiva found")
```

Comment:
Why do we need this?

Reply:
Because there are other areas that are surrounded by the Kansas City area that are not exactly Kansas City. Some of these areas are Galdstone, Raytown and, confusingly enough, North Kansas City.